htdocs/core/boxes/box_services_contracts.php

```diff
@@ -96,12 +96,12 @@ function loadBox($max=5)
 					$contratlignestatic->id=$objp->cdid;
 					$contratlignestatic->fk_contrat=$objp->rowid;
 					$contratlignestatic->label=$objp->label;
-					$contratlignestatic->description=$objp->description;
+					$contratlignestatic->description=dol_trunc($objp->description);
```

Comment:
When a td has an overflow proerty (automatic truncation), then the max-width is in fact the min-width. It looks strange, but it is. The behaviour is
it there is enough space, we show all
if not, it is truncated.
It is what you planned to do with the truncation (truncated when there is no enough room) ?